src/bfx_tools/vaxrank.ml

```diff
@@ -0,0 +1,150 @@
+open Biokepi_run_environment
+open Common
+
+
+(*
+  $ vaxrank --help
+  usage: vaxrank [-h] --vcf VCF [--genome GENOME] --bam BAM
```

Comment:
AFAIK, this is still in the works. The variants parsing functionality comes from `isovar` and it doesn't seem to support multiple VCF at the moment:
https://github.com/hammerlab/isovar/blob/master/isovar/args/variants.py#L26